crates/egui/src/response.rs

```diff
@@ -6,6 +6,41 @@ use crate::{
 
 // ----------------------------------------------------------------------------
 
+bitflags::bitflags! {
+    /// The state of a widget.
```

Comment:
I have no idea what would be good docs for this struct. Feedback would be appreciated :)